packages/babel-plugin-transform-parameters/src/params.js

```diff
@@ -32,7 +32,11 @@ function isSafeBinding(scope, node) {
 const iifeVisitor = {
   ReferencedIdentifier(path, state) {
     const { scope, node } = path;
-    if (node.name === "eval" || !isSafeBinding(scope, node)) {
+    if (
+      node.name === "eval" ||
+      !isSafeBinding(scope, node) ||
```

Comment:
This should be considered a workaround hotfix because the ReferencedIdentifier in params initializers should point to the parent of the function scope. Because `path.scope` is not accurate here, we have to do a hotfix and check `state.scope` again.

The ideal fix should construct a separate scope for parameter initializer.